Hyper/Controllers/BotsController.cs

```diff
@@ -10,35 +10,36 @@ internal class BotsController : IController, IInputSubscriber
 {
     private readonly Scene _scene;
 
-    private readonly ModelShader _shader;
+    private readonly AbstractModelShader _modelShader;
+
+    private readonly AbstractObjectShader _objectShader;
 
-    private readonly ObjectShader _objectShader;
-    
     private float _elapsedSeconds = 0;
-    
+
     private bool _showBoundingBoxes = false;
 
-    public BotsController(Scene scene, Context context, ModelShader shader, ObjectShader objectShader)
+    public BotsController(Scene scene, Context context, AbstractModelShader modelShader, AbstractObjectShader objectShader)
```

Comment:
Shouldn't this also use the teleporter?

Reply:
It should. The same is true for `VehiclesController`. But I thought we could add it once bots and vehicles are "more developed"